custom/ewsghana/filters.py

```diff
@@ -97,8 +100,92 @@ def options(self):
             .order_by('name')
 
 
-class EWSLocationFilter(AsyncLocationFilter):
-    template = "reports/filters/location_async.html"
+def load_locs_json(domain, selected_loc_id=None, include_archived=False, show_admin=True,
```

Comment:
can you clarify/add a comment to the code about what's different here from https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/locations/util.py#L18?

also, depending on the answer, anyway to parameterize the change rather than copy/pasting the whole function?